dirsrvtests/tests/suites/ds_logs/ds_logs_test.py

```diff
@@ -839,21 +840,103 @@ def test_etime_order_of_magnitude(topology_st, clean_access_logs, remove_users,
     assert len(result_str) > 0
 
     # The result_str returned looks like :
-    # [23/Apr/2020:06:06:14.366429900 -0400] conn=1 op=93 RESULT err=0 tag=101 nentries=30 etime=0.005723017
+    # For ds older than 1.4.3.8: [23/Apr/2020:06:06:14.366429900 -0400] conn=1 op=93 RESULT err=0 tag=101 nentries=30 etime=0.005723017
+    # For ds newer than 1.4.3.8: [21/Oct/2020:09:27:50.095209871 -0400] conn=1 op=96 RESULT err=0 tag=101 nentries=30 wtime=0.000412584 optime=0.005428971 etime=0.005836077
     
     log.info('get the operation end time from the RESULT string')
     # Here we are getting the sec.nanosec part of the date, '14.366429900' in the above example
     end_time = (result_str.split()[0]).split(':')[3]
 
     log.info('get the logged etime for the operation from the RESULT string')
     # Here we are getting the etime value, '0.005723017' in the example above
-    etime = result_str.split()[8].split('=')[1][:-3]
+    if ds_is_older('1.4.3.8'):
+        etime = result_str.split()[8].split('=')[1][:-3]
+    else:
+        etime = result_str.split()[10].split('=')[1][:-3]
 
     log.info('Calculate the ratio between logged etime for the operation and elapsed time from its start time to its end time - should be around 1')
     etime_ratio = (Decimal(end_time) - Decimal(start_time)) // Decimal(etime)
     assert etime_ratio <= 1
 
 
+@pytest.mark.skipif(ds_is_older('1.4.3.8'), reason="Fail because of bug 1850275")
+@pytest.mark.bz1850275
+def test_optime_and_wtime_keywords(topology_st, clean_access_logs, remove_users, disable_access_log_buffering):
+    """Test that the new optime and wtime keywords are present in the access log and have correct values
+
+    :id: dfb4a49d-1cfc-400e-ba43-c107f58d62cf
+    :setup: Standalone instance
+    :steps:
+         1. Unset log buffering for the access log
+         2. Delete potential existing access logs
+         3. Add users
+         4. Search users
+         5. Parse the access log looking for the SRCH operation log
+         6. From the SRCH string get the op number of the operation
+         7. From the op num find the associated RESULT string in the access log
+         8. Search for the wtime optime keywords in the RESULT string
+         9. From the RESULT string get the wtime, optime and etime values for the operation 
+         10. Check that optime + wtime is approximatively etime
+    :expectedresults:
+         1. access log buffering is off
+         2. Previously existing access logs are deleted
+         3. Users are successfully added
+         4. Search operation is successful
+         5. SRCH operation log string is catched
+         6. op number is collected
+         7. RESULT string is catched from the access log
+         8. wtime and optime keywords are collected
+         9. wtime, optime and etime values are collected
+         10. (optime + wtime) =~ etime
+    """
+
+    entry = DSLdapObject(topology_st.standalone, DEFAULT_SUFFIX)
```

Comment:
Generally, you can use 'Domain' class for the root suffix but I don't see how 'entry' object is used in this code at all...

Reply:
Right, the entry object is not used. I did a small change to remove the line in both tests.
@droideck could you please review again ? Thanks !